### ISSUE TITLE
Is it possible to create polygon holding the point (circle)

### ISSUE THREAD

**Issue by chandandas** [taqimustafa/fabricjs-polygon#5]
Hi,

Thanks for this code. However i want to create polygon using pre-defined points. Then i want to hold a particular point and then begin dragging it and then only the ploygon should be created.

Is it possible?

**Comment by taqimustafa** [comment]
@chandandas Yes, it might be possible if we add circles on each point of polygon, and then when ever that circle is dragged, update the polygon array. It will be similar to creating Fabricjs controls, but instead of 4 on the corners, it will be on each polygon corner

**Comment by chandandas** [comment]
Ok. Thanks. Will try though not that great in fabricjs!
